tests/e2e/simple-app.test.ts

```diff
@@ -12,7 +12,9 @@ test('Renders the Home page correctly', async ({ page, simple }) => {
 
   await expect(page).toHaveTitle('Simple Next App')
 
-  expect(headers['cache-status']).toBe('"Next.js"; hit\n"Netlify Edge"; fwd=miss')
+  expect(headers['cache-status']).toBe(
+    '"Next.js"; hit\n"Netlify Durable"; fwd=miss\n"Netlify Edge"; fwd=miss',
+  )
```

Comment:
duplicated from https://github.com/netlify/next-runtime/pull/2527 to unblock myself